tests/cases/compiler/moduleResolutionWithSuffixes_threeLastIsBlank3.ts

```diff
@@ -0,0 +1,15 @@
+// moduleSuffixes has three entries, and the last one is blank. Module resolution should match on the blank suffix.
+
+// @filename: /tsconfig.json
+{
+	"compilerOptions": {
+		"moduleResolution": "node",
+		"traceResolution": true,
+		"moduleSuffixes": ["-ios", "__native", ""]
+	}
+}
+
+// @filename: /index.ts
+import { base } from "./foo";

```

Comment:
Can we add tests that use module specifiers in the forms:

- `"./foo.js"`
- `"./foo"` where `foo` is a folder with `Index.*.ts`
- `"some-library"`
- `"some-library/foo"`
- maybe something that uses `"paths"` in compilerOptions?

Is the logic supposed to apply for imports from node_modules?

Reply:
Added more tests to cover these scenarios.